.env.sample

```diff
@@ -40,3 +40,11 @@ CERTIFICATE_SERVICE_URL = http://registry-service:8081
 
 PROJECT_CERTIFICATE_ON_OFF = "ON/OFF"                                           // Project certificate enable or disable flag
 
+USER_DELETE_TOPIC = {{env_name}}.delete.user                                    // Topic name for user delete event consumer
```

Comment:
@ankitpws - Let's make this dependent and configurable ON/OFF based

Reply:
made